syslog/lang/en

```diff
@@ -1,4 +1,4 @@
-index_title=System Logs
+index_title=System Logs RS
```

Comment:
What is "RS" ?

Reply:
RS represents Rsyslog in more specific way, e.g.:

<img width="192" alt="image" src="https://github.com/webmin/webmin/assets/4426533/4ac83694-2a72-4912-b971-f6fd9698ab56">

While more general and widely available `systemd` will use title **System Logs**.

Reply:
Not all syslog versions are rsyslog though

Reply:
Yeah, the other will display as **System Logs NG**.

Reply:
Let's just leave the `syslog` module as **System Logs**

Reply:
Please, let's not. The `syslog` module is being replaced with `journal` and is no longer installed and recommended by many distros we support, in particular with Virtualmin!

**RSyslog** is the only well-supported syslog version out there nowadays.

I think it would be better to classify names of log modules this way. Old users can always change the name to whatever they wish in the Webmin Configuration page, whereas Virtualmin users won't be confused with the  module name that they will actually use.

Reply:
Maybe just call them RSyslog and Syslog NG then?

Reply:
I really think it's easier to keep it simple.

Reply:
I don't think anyone is going to know what RS stands for though!

Reply:
It's somewhat new but expected, as RSyslog uses it as part of the name and in the logo too. It also goes along well with Syslog-ng.